yarn_api_client/hadoop_conf.py

```diff
@@ -1,21 +1,64 @@
 # -*- coding: utf-8 -*-
 import os
 import xml.etree.ElementTree as ET
+try:
+    from httplib import HTTPConnection, OK
+except ImportError:
+    from http.client import HTTPConnection, OK
 
 CONF_DIR = '/etc/hadoop/conf'
 
 
-def get_resource_manager_host_port():
-    config_path = os.path.join(CONF_DIR, 'yarn-site.xml')
+def get_rm_ids(hadoop_conf_path):
+    rm_ids = parse(os.path.join(hadoop_conf_path, 'yarn-site.xml'), 'yarn.resourcemanager.ha.rm-ids')
+    if rm_ids is not None:
+        rm_ids = rm_ids.split(',')
+    return rm_ids
+
+
+def get_resource_manager(hadoop_conf_path, rm_id = None):
```

Comment:
If `get_resource_manager` is a function for internal usage - prefix it with underscore.